manager/backupbackingimage.go

```diff
@@ -0,0 +1,70 @@
+package manager
+
+import (
+	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
+	"github.com/longhorn/longhorn-manager/util"
+	"github.com/pkg/errors"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (m *VolumeManager) ListBackupBackingImagesSorted() ([]*longhorn.BackupBackingImage, error) {
+	backupBackingImageMap, err := m.ds.ListBackupBackingImages()
```

Comment:
`ListBackupBackingImagesRO`

Reply:
I need the map, because we need to sort the list.
Using RO will only return list and I still need to deep copy the item when sorting.

The implementation is the same as other APIs
https://github.com/longhorn/longhorn-manager/blob/8cdf6820620161a0e2b1e71f2994a3c4f763917c/manager/volume.go#L66-L81

```
func (m *VolumeManager) ListSorted() ([]*longhorn.Volume, error) {
	volumeMap, err := m.List()
	if err != nil {
		return []*longhorn.Volume{}, err
	}

	volumes := make([]*longhorn.Volume, len(volumeMap))
	volumeNames, err := util.SortKeys(volumeMap)
	if err != nil {
		return []*longhorn.Volume{}, err
	}
	for i, volumeName := range volumeNames {
		volumes[i] = volumeMap[volumeName]
	}
	return volumes, nil
}
```